pkg/sdk/external_functions_impl_gen.go

```diff
@@ -32,12 +33,37 @@ func (v *externalFunctions) Show(ctx context.Context, request *ShowExternalFunct
 	return resultList, nil
 }
 
-func (v *externalFunctions) ShowByID(ctx context.Context, id SchemaObjectIdentifier) (*ExternalFunction, error) {
+func (v *externalFunctions) ShowByID(ctx context.Context, id SchemaObjectIdentifier, arguments []DataType) (*ExternalFunction, error) {
 	externalFunctions, err := v.Show(ctx, NewShowExternalFunctionRequest().WithLike(&Like{Pattern: String(id.Name())}))
 	if err != nil {
 		return nil, err
 	}
-	return collections.FindOne(externalFunctions, func(r ExternalFunction) bool { return r.Name == id.Name() })
+	return collections.FindOne(externalFunctions, func(r ExternalFunction) bool {
+		database := strings.Trim(r.CatalogName, `"`)
```

Comment:
CatalogName and SchemaName should be nullable. According to the documentation, they can be null for built-in functions.